src/streaming-services/disneyplus/DisneyplusParser.ts

```diff
@@ -20,44 +17,21 @@ class _DisneyplusParser implements ScrobbleParser {
 		this.id = '';
 		this.videoId = '';
 		this.progress = 0.0;
-		this.isPaused = false;
+		this.isPaused = true;
 	}
 
-	parseSession = (): DisneyplusSession => {
-		const loadingSpinner =
-			document.querySelector('.overlay__loading:not([style="display: none;"])') !== null;
-		const pauseIcon = document.querySelector('.pause-icon') !== null;
-		const playIcon = document.querySelector('.play-icon') !== null;
-
-		if (loadingSpinner) {
-			this.isPaused = true;
-		} else if (playIcon && !pauseIcon) {
-			this.isPaused = true;
-		} else if (pauseIcon && !playIcon) {
-			this.isPaused = false;
-		}
-
-		const paused = this.isPaused;
-		const progress = this.parseProgress();
-		return { paused, progress };
+	getVideoElement = (): HTMLVideoElement => {
```

Comment:
Any particular reason why you put this in a separate function? Isn't it just a waste since it can be inlined?

Reply:
Nah. Forgot to clean up. Started with just keeping the names identical. Didn't realise it was just used once. Will fix!